swap/src/bin/swap.rs

```diff
@@ -36,21 +43,33 @@ extern crate prettytable;
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    init_tracing(LevelFilter::Trace).expect("initialize tracing");
```

Comment:
@rishflab @D4nte I am using tracing for user output at the moment. I wonder if that is what we want, or if we should distinguish between user-output (`println`) and logging (`tracing`). Strong opinions anyone?

Reply:
A good way to think about stdout vs stderr is that `stdout` should play nice with piping and hence be "machine-readable" vs `stderr` is for humans and hence can contain all the diagnostic information.

If your CLI doesn't have anything that it could feed to another CLI, probably everything should go on stderr :)